extensions/geoCoordinatesExtension/schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "More Info Link",
+  "description": "An extension allowing for the addition of the geographic location where an individual earned a badge.",
+  "type": "object",
+  "properties": {
+    "description": {
+      "type": "string",
+      "format": "text"
```

Comment:
"format" is a weird part of JSON-schema. See [this explanation](http://spacetelescope.github.io/understanding-json-schema/reference/string.html#format). It's not required to be supported by implementations.